FR/2DCode/raw/vaccin_ok.json

```diff
@@ -24,16 +24,16 @@
     }
```

Comment:
Value of ci should be dgci:V1:FR:AP6D4CPJVI1RHAHF6OBPQM0CG:84

Reply:
fix ok